bedrock/base/templates/includes/news-feed.html

```diff
@@ -24,7 +24,7 @@ <h3 class="entry-title">{{ article.blog_title }}</h3>
               <p>{{ article.htmlify() }}</p>
             </div>
             <p class="entry-cta">
-              <a href="{{ article.link }}?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=fx-content-hub" class="cta-link" data-link-name="{{ article.blog_title }}" data-link-type="link" data-link-position="Fresh from the blog">{{ article.blog_name }}</a>
+              <a href="{{ article.link }}?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=fx-content-hub" class="cta-link" rel="nofollow" data-link-name="{{ article.blog_title }}" data-link-type="link" data-link-position="Fresh from the blog">{{ _('Read More') }}</a>
```

Comment:
Nit: should this just be `Read more` (lowercase m)? This seems consistent with casing for other cta links on the hub pages.

Also, should we also change `data-link-name="{{ article.blog_title }}"` to `data-link-name="Read more"`? The value for this data attribute should generally match the link text.